lib/ResponseDefinitions.php

```diff
@@ -197,18 +197,21 @@
  *     optionId: int,
  *  }
  *
- * @psalm-type TalkPoll = array{
+ * @psalm-type TalkPollDraft = array{
  *     actorDisplayName: string,
  *     actorId: string,
  *     actorType: string,
- *     details?: TalkPollVote[],
  *     id: int,
  *     maxVotes: int,
- *     numVoters?: int,
  *     options: string[],
  *     question: string,
  *     resultMode: int,
  *     status: int,
```

Comment:
Possible to do?
```suggestion
 *     resultMode: 0|1,
 *     status: 0|1|2,
```

Reply:
- I attempted it in #12783 but it was too much to finish. Maybe I can start with a single place instead, so will have another look tomorrow